sources/packages/web/src/services/http/DesignationAgreementApi.ts

```diff
@@ -0,0 +1,38 @@
+import HttpBaseClient from "@/services/http/common/HttpBaseClient";
+import {
+  SubmitDesignationAgreementDto,
+  GetDesignationAgreementDto,
+} from "@/types/contracts/DesignationAgreementContract";
+
+/**
+ * Http API client for Designation agreements.
+ */
+export class DesignationAgreementApi extends HttpBaseClient {
+  async submitDesignationAgreement(
+    designationAgreement: SubmitDesignationAgreementDto,
+  ): Promise<void> {
+    try {
+      await this.apiClient.post(
```

Comment:
we have `.postCall()`
![image](https://user-images.githubusercontent.com/77353155/152599703-3ad58bf6-e868-4bb8-af53-10ccf243a6ee.png)